PeerioTechnologies/peerio-desktop
# fix: quit first instance is second is running from DMG

#### Relevant info and issue/PR links

https://app.clubhouse.io/peerio/story/3624/mac-inform-users-that-they-should-close-another-copy-if-it-s-running-before-moving-to-applications-folder-from-dmg

#### Testing instructions

Any OS (one of Mac & Windows or Linux is fine):

I.

1. Launch the app (not from DMG).
2. Launch another instance of app (not from DMG). Make sure another instance won't launch, and instead make the first instance window appear, as usual.

Mac only:

II.

1. Launch the app (not from DMG).
2. Mount DMG with the app (`npm run internal-build-mac`, in dist), and launch the app from it.
3. Check that the **first** copy quits and the new one runs showing "Peerio 2 is running from a disk image..." message.

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
